lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -66,72 +64,39 @@ def generate
             'SWIFT_INCLUDE_PATHS' => '$(inherited) ',
           }.merge(embedded_content_settings)
 
-          @xcconfig = Xcodeproj::Config.new(config)
```

Comment:
removes state from this generator, there is no need to hold on to the `xcconfig` to this instance after we have doen the work.